serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -11,8 +11,11 @@
 //** xref:core/state-data-filtering.adoc[State data filtering]
 //** xref:core/defining-an-input-schema-for-workflows.adoc[Defining an input schema for your workflows]
 ** xref:core/custom-functions-support.adoc[Custom functions for your Serverless Workflow service]
-//* Tooling
-//** xref:tooling/tooling.adoc[Tooling]
+* Tooling
+** Kogito Serverless Workflow Tools - Quarkus Dev UI
```

Comment:
I think since this item is already under `Kogito Serverless Workflow`, we could omit that prefix in our sections.

```suggestion
** Quarkus Dev UI Tools
```